ls1intum/Artemis
# [Bugfix] Fix programming language Assembler translation

### Checklist

### Motivation and Context


The "Programming Language" section of the programming exercise detail page displayed a "missing translations" error for Assembler.

### Description

Renamed the translation keys so that the translations can be loaded.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Generate an Assembler programming exercise
4. Navigate to the exercise's detail page and check that both English and German the translations are loaded
5. (Bonus) Check that the translations are loaded for each programming language

### Test Coverage



Nothing changed
